app/great_battle.py

```diff
@@ -0,0 +1,16 @@
+from app.structuring_the_knight_personality import Knight
+
+
+class BattleOfTwoKnight:
```

Comment:
Pick up the class name without prepositions, the class name should be as laconic as possible.

Reply:
👍
![just_be_creative](https://github.com/mate-academy/py-knights-fighting/assets/108234997/33ba2df4-e516-4c64-a9cb-15bb3727ae0c)